libpod/container_internal.go

```diff
@@ -220,11 +232,22 @@ func (c *Container) teardownStorage() error {
 	if err := os.RemoveAll(artifacts); err != nil {
 		return errors.Wrapf(err, "error removing artifacts %q", artifacts)
 	}
+	if c.config.UserNsCtr != "" {
```

Comment:
This seems very wrong. UserNsCtr is the ID of the container whose user namespace we are using. It should NOT be a dir on the host...

Reply:
yes true, not sure how that ended up in the code.  Dropping it